app/templates/media/show.hbs

```diff
@@ -43,7 +43,7 @@
 
 <div class="container">
   {{! sidebar }}
-  <div class="is-sticky media-sidebar--sticky">
+  <div class="is-sticky media-sidebar--sticky" data-margin-top="260">
```

Comment:
I have no idea why but this sticky doesn't work consistently, while the rest do. Maybe has to do with the weird positioning we're doing on it?